src/staticanalysis/bot/static_analysis_bot/tasks/clang_tidy.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
```

Comment:
I wonder why this and clang_format.py show as new files :/

Reply:
Maybe because i renamed from `clang/format.py` to `tasks/clang_format.py` and the levenshtein distance is too big :man_cartwheeling:

Reply:
It's a pity that we lose the history, but well...